docker/docker-compose-build-all.yml

```diff
@@ -15,8 +15,6 @@ services:
     extends:
       service: gradle
       file: docker-compose-gradle.yml
-    ports:
```

Comment:
Is this a mistake? Please revert it back. And Line 32, 33.

Reply:
We don't need ports snippet in this build-all yml file , ok I can revert to remove confusion.

Reply:
If we have this, we getting error while running SCXA containers in my Laptop at least.